Makefile

```diff
@@ -1829,7 +1832,9 @@ INCLUDE+=\
 -I$(ESP_APP_TEMPLATE_PATH)/components/arduino-esp32/cores/esp32 \
 -Itargets/esp32/include
 LDFLAGS+=-nostdlib -u call_user_start_cpu0 -Wl,--gc-sections -Wl,-static -Wl,-EL
-LIBS+=-T esp32_out.ld \
+#LIBS+=-T$(ESP_IDF_PATH)/components/esp32/ld/esp32_out.ld \
+#LIBS+=-T$(ESP_APP_TEMPLATE_PATH)/components/arduino-esp32/tools/sdk/ld/esp32_out.ld \
+LIBS+=-T$(ESP_APP_TEMPLATE_PATH)/build/esp32/esp32_out.ld \
```

Comment:
I need to get add the path here to get a compile without missing symbols, however have and issue with:
/opt/xtensa-esp32-elf/bin/../lib/gcc/xtensa-esp32-elf/4.8.5/../../../../xtensa-esp32-elf/bin/ld: warning: cannot find entry symbol _start; not setting start address